### ISSUE TITLE
Wikiteam sync is not finding large number of sites

### ISSUE THREAD

**Issue by nemobis** [WikiApiary/WikiApiary#130]
[WikiTeam websites](https://wikiapiary.com/wiki/WikiTeam_websites) claims there are 7136 unarchived wikis. I took a [sample of 5500](http://koti.kapsi.fi/~federico/tmp/bee.txt) and compared it to the [list of known done dumps](https://code.google.com/p/wikiteam/source/browse/trunk/batchdownload/taskforce/mediawikis_done_2014.txt) (dumps produced locally and whose hash is found on an archive.org item).

A simple comm -12 finds that 2819/5500 wikis actually were downloaded; if I also remove `https?://` and `/(index|api).php` to account for inconsistencies, that becomes 3605/5500 or 65 %. I [uploaded the list of those domains](http://koti.kapsi.fi/~federico/tmp/bee-done.txt).

**Comment by nemobis** [comment]
Can I help fix this bug? It helps WikiTeam's work most of all, I feel, so it's just right that we help fix it. If I understand correctly, the code should be in apiary/tasks/farmer/ , but the directory is empty in all branches. If there's a way I can see the code and test what's going wrong for those URLs, please let me know.

**Comment by emijrp** [comment]
Please, share the bot code so I can check why it is missing many wikis.

**Comment by jthingelstad** [comment]
_Moved to WikiApiary/FarmerBee#3._

**Comment by nemobis** [comment]
> Moved to WikiApiary/FarmerBee#3.

Is this repository/issue tracker private?

**Comment by jthingelstad** [comment]
@nemobis Hello nemo… the repo that was at WikiApiary/FarmerBee didn’t get any traction. It had no code and only a handful of issues. Since there was nothing there I deleted the repo.

It was definitely my idea that bots that harvest wiki farms should be totally separate from the other bots that run WikiApiary. This idea never got enough momentum to actually get built though.

tl;dr: It’s not private, the repo was deleted.

/cc @hexmode

**Comment by nemobis** [comment]
Jamie Thingelstad, 07/09/2016 05:27:

> It was definitely my idea that bots that harvest wiki farms should be
> totally separate from the other bots that run WikiApiary. This idea
> never got enough momentum to actually get built though.

Thanks. Do you plan to still run the bot? Should someone willing to help 
with the sync just rewrite from scratch?

**Comment by jthingelstad** [comment]
@nemobis the MediaWiki Stakeholders group have super graciously offered to run WikiApiary going forward, led by @hexmode. I _believe_ that the WikiTeam bot still runs weekly, but honestly I’m not sure.

I know the version I wrote was particularly not great. To the extent that WikiTeam wanted to write a bot that ran on it’s own I think that would probably be better, but ultimately up to you all.

**Comment by hexmode** [comment]
@thingles I'm looking over cron jobs and such.  Would this one run under your user?

**Comment by emijrp** [comment]
Some months ago I wrote a new bot for updating IA dump links in WikiApiary.

Here is the code https://github.com/WikiTeam/wikiteam/tree/master/wikiapiary

It can be launched by anyone with basic knowledge of pywikibot.

2016-09-08 2:33 GMT+02:00 Mark A. Hershberger notifications@github.com:

**Comment by jthingelstad** [comment]
@hexmode yes. But I just looked and they are commented out so I’m thinking you moved them. 👍

**Comment by hexmode** [comment]
@emijrp could you create a pull request against the [WikiApiary/Wikibees](https://github.com/WikiApiary/wikibees) repository for the script?  Also helpful would be the frequency you think the script should run at.
